src/assets/locales/de/servicerelease.json

```diff
@@ -120,6 +127,14 @@
     "submenuNotAvailable": "Sub-Menu is not available for not active apps.",
     "sortOptions": {
       "deactivate": "Deactivate"
+    },
+    "serviceDeactivate": {
+      "headerTitle": "Deactivate Service",
+      "description": "Deactivation of service will deactivate the service offer inside the Catena-X marketplace. Any current active subscription will stay active and customers wont be effected by the deactivation. New subscriptions are not possible.",
+      "checkboxLabel": "Hereby I confirm that I have read the business service terms and conditions and that I am deactivating my company business services from the marketplace with immediate effect. I understand that it is not possible to self-reverse the deactivation.",
+      "checkboxErrorMsg": "Please confirm that you have read the deactivation impact above before proceeding",
+      "successMsg": "Deactivation successfully completed",
+      "errorMsg": "Error! Something went wrong"
```

Comment:
same here - translation needed to reduce future efforts

Reply:
Translated using https://translate.google.com/